File: tooling/hibernate-gradle-plugin/src/test/resources/projects/multi-part-source-set-name/src/mySpecialSourceSet/java/TheEmbeddable.java
import jakarta.persistence.Embeddable;

@Embeddable
public class TheEmbeddable {
	private String valueOne;
	private String valueTwo;

	public String getValueOne() {
		return valueOne;
	}

	public void setValueOne(String valueOne) {
		this.valueOne = valueOne;
	}

	public String getValueTwo() {
		return valueTwo;
	}

	public void setValueTwo(String valueTwo) {
		this.valueTwo = valueTwo;
	}
}

File: tooling/hibernate-gradle-plugin/src/test/resources/projects/simple-kotlin/src/main/kotlin/TheEmbeddable.kt
/*
 * SPDX-License-Identifier: Apache-2.0
 * Copyright Red Hat Inc. and Hibernate Authors
 */
import jakarta.persistence.Embeddable;

@Embeddable
class TheEmbeddable {
	var valueOne: String? = null
	var valueTwo: String? = null
}
